numpy/lib/nanfunctions.py

```diff
@@ -219,6 +218,12 @@ def nanmin(a, axis=None, out=None, keepdims=np._NoValue):
         does not implement `keepdims` any exceptions will be raised.
 
         .. versionadded:: 1.8.0
+    initializer : scalar, optional
+        The maximum value of an output element. Must be present to allow
```

Comment:
See comment above.

Reply:
Actually, this one is correct. It's for `nanmin`, so it IS the maximum value.

Reply:
This mostly referred to the initial comment of asking for a rewording. But I then later realized I did not have a good concrete suggestion... Anyway, at a minimum, just fix the wrong one for maximum above.

Reply:
> Anyway, at a minimum

Unhelpful choice of words...